common/build.gradle

```diff
@@ -100,6 +100,6 @@ dependencies {
 	api 'com.google.zxing:core:3.4.1'
 
 	testImplementation 'junit:junit:4.+'
```

Comment:
Could we change this to a fixed version? Not that it should be a problem with JUnit, but in general I prefer not having unstable dependencies (and also getting rid of the Android Studio lint warning would be nice :smile: )
Do we even need it here? We don't have any unit tests in the common module and the wallet has its own testing dependencies